src/spawn.c

```diff
@@ -67,6 +67,7 @@
 #else
 # include "support.h"
 #endif
+#include "utils.h"
```

Comment:
870c1379a0f633195c74e73a839d83dc3e50abfa fixes this (it's in #1300, but can be added here too if wanted)